helm/cluster-cloud-director/values.schema.json

```
@@ -77,6 +77,65 @@
             "examples": [
                 30
             ]
+        },
+        "registryConfigs": {
+            "type": "array",
+            "title": "Endpoint Credentials of Container Registies",
```

Converting to sentence case, and adding an "r"

```suggestion
            "title": "Endpoint credentials of container registries",
```